lib/voxpupuli/release/rake_tasks.rb

```diff
@@ -52,3 +52,125 @@
     fail "Unable to find a CHANGELOG.md entry for the #{v} release."
   end
 end
+
+namespace :release do
+  desc "Prepare a release"
+  task prepare: ['release:porcelain:changelog'] do
+    v = Blacksmith::Modulefile.new.version
+    Rake::Task["strings:generate:reference"].invoke if File.exist?('REFERENCE.md')
+    puts <<~MESSAGE
+
+      Please review these changes and commit them to a new branch:
+
+          git checkout -b release-#{v}
+          git commit -m "Release #{v}"
+
+      Then open a Pull-Request and wait for it to be reviewed and merged).
+    MESSAGE
+  end
+
+  begin
+    require 'github_changelog_generator/task'
+    require 'puppet_blacksmith'
+  rescue LoadError
+    namespace :porcelain do
```

Comment:
Technically you can pull the `namespace` declaration` outside of the begin/rescue` block since it's repeated in both branches.